python/tests/spark/sql/codegen/test_mlflow_logger.py

```diff
@@ -40,3 +42,120 @@ def test_none_value_tags(tmp_path: Path, resnet_model_uri):
         assert tags[rikai.mlflow.CONF_MLFLOW_OUTPUT_SCHEMA] == "output_schema"
         assert rikai.mlflow.CONF_MLFLOW_PRE_PROCESSING not in tags
         assert rikai.mlflow.CONF_MLFLOW_POST_PROCESSING not in tags
+
+
+def test_model_and_version_tags(tmp_path: Path, resnet_model_uri):
```

Comment:
this is one huge test case, can we break this into smaller pieces?

Reply:
for example, you can factor out the model assertions into a function and call it repeatedly.